examples/analytics-iframe-transport-remote-frame.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <title>Requests Frame</title>
     <script>
+      console.log('Created Iframe!!!!!!');
```

Comment:
remove

Reply:
I added this intentionally to inform that the iframe has loaded w/o looking at network request : ) 

Anyhow I think I could change my example page to not use transport: iframe. I'll remove that